setup.cfg

```diff
@@ -5,7 +5,7 @@ description-file = Readme.md
 test=pytest
 
 [tool:pytest]
-python_files = tests/unit/*.py tests/integration/*.py 
+python_files = tests/unit/*.py tests/integration/*.py tests/functional/*.py
```

Comment:
Fair for now, but I think we might want to think about defining sets of fast and slow tests at some point.